lib/vdsm/tool/configurators/multipath.py

```diff
@@ -214,6 +214,7 @@
 # We blacklist following protocols:
```

Comment:
Please update the commit message, this patch does not change
mpathconf.py.

Reply:
I guess we can agree on some "PR best practices".
Like we can continue using prefixes (`core`, `webadmin`, etc.) in commits that form the PR, but PR title can be a single 1-sentence starting with uppercase describing what it does **without** any prefix.
In this case it can look like: `Add a blacklist for rbd devices`.

Reply:
Ack